dalle2_pytorch/dalle2_pytorch.py

```diff
@@ -32,6 +34,38 @@
 from x_clip import CLIP
 from coca_pytorch import CoCa
 
+# Diffusion Prior model loading and saving functions
+
+def load_diffusion_model(dprior_path, device ):
+
+        dprior_path = Path(dprior_path)
+        assert dprior_path.exists(), 'Dprior model file does not exist'
+        loaded_obj = torch.load(str(dprior_path), map_location='cpu')
+
+        # Get hyperparameters of loaded model 
+        dpn_config = loaded_obj['hyperpmts']['Diffusion_Prior_Network']
```

Comment:
Let's call it hparams and diffusion_prior_network

Reply:
fixed